packages/contracts-bedrock/deploy-config/hardhat.json

```diff
@@ -47,7 +47,7 @@
   "faultGameMaxDepth": 8,
   "faultGameMaxDuration": 2400,
   "faultGameGenesisBlock": 0,
-  "faultGameGenesisOutputRoot": "0x0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
why `0x00...01`?

Reply:
Can be anything as long as it's non-zero since that means the anchor state is uninitialized.

Reply:
Made this deadbeef to make it more clear this is an arbitrary non-zero value.